src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -764,6 +764,27 @@
     text: |
         az iot hub update --name MyIotHub --set properties.allocationPolicy="GeoLatency"
     crafted: true
+  - name: Update units of an IoT Hub to 2
+    text: >
+        az iot hub update -n MyIotHub --unit 2
+  - name: Update pricing tier for an IoT Hub as S2
+    text: >
+        az iot hub update -n MyIotHub --sku S2
+  - name: Update the retention days of an IoT Hub device-to-cloud events as 3 days
```

Comment:
nit, Would remove 'an' i.e. `Update the retention days of IoT Hub device-to-cloud events as 3 days`

Reply:
Done